templates/overrides/search/form--header--oe-whitelabel-search-form.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Search form template for the header region.
+ *
+ * @see ./core/modules/system/templates/form.html.twig
+ */
+#}
+<div class="bg-lighter py-4 py-lg-3">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-lg-6 offset-lg-3">
+        <form {{ attributes.addClass('bcl-search-form', 'submittable') }}>
+          <div class="bcl-search-form__group">
+            {{ children }}
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Does all of this really belong into the form template?
For me it would belong into the page template or perhaps in the block template.
Also, in the new version in netlify, we now have a page title in the same region. I know we don't want to implement the latest changes in a ticket that is already in progress, but it does validate the idea that the outer wrapper divs should not really be part of the search form.